api/controllers/users.py

```diff
@@ -465,3 +467,44 @@ def upload_user_profile_picture(credentials, id):
     except Exception as ex:
         logger.exception("Could not upload user profile picture: %s" % (ex))
         bottle.abort(400, "Could not upload user profile picture")
+
+
+@bottle.post("/users/model/upload")
+@_auth.requires_auth
+def model_upload_s3_dynalab_2(credentials):
+    upload = bottle.request.files.get("file")
+    file_name = bottle.request.forms.get("file_name")
+    file_type = bottle.request.forms.get("file_type")
+    user_name = bottle.request.forms.get("user_name")
+    user_id = bottle.request.forms.get("user_id")
+    task_code = bottle.request.forms.get("task_code")
+
+    session = boto3.Session(
+        aws_access_key_id=config_file["aws_access_key_id"],
+        aws_secret_access_key=config_file["aws_secret_access_key"],
+        region_name=config_file["aws_region"],
+    )
+    s3_service = session.client("s3")
+    sqs_service = session.client("sqs")
+
+    if file_type != "application/zip":
```

Comment:
Does this need more checks, eg with a reasonable size limit etc?

Reply:
good point, let me add a size limit checker